.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  NODE_VERSION: 14
```

Comment:
We'll probably want to bump the Node version in the `package.json` file, too.

Might be better to do that in a separate PR that lands before this one (since that will be a breaking change).

Reply:
I made that change and split out a dedicated commit. I tried to land this in a separate branch and couldn't get things to build.

Reply:
That's fair. I'm comfortable with the changes in that case since we want to get back into support.